lib/view/taskView.dart

```diff
@@ -221,6 +170,59 @@ class _taskViewState extends State<taskView> {
     );
   }
 
+  void _showTaskDetailsDialog(List<Task> tasks) {
+    showDialog(
+      context: scaffoldKey.currentState!.context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          title: const Text('Task Details'),
+          content: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: tasks.map((task) {
+              return Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text('Task ID: ${task.id}'),
+                  Text('Name: ${task.name}'),
+                  Text('Description: ${task.description}'),
+                  const Divider(),
+                ],
+              );
+            }).toList(),
+          ),
+          actions: [
+            TextButton(
+              onPressed: () {
+                Navigator.of(context).pop();
+              },
+              child: const Text('Close'),
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  void _showTaskNotFoundDialog() {
+    showDialog(
+      context: scaffoldKey.currentState!.context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          title: const Text('Task Not Found'),
+          content: const Text('The task with ID was not found.'),
```

Comment:
maybe add the ID that was searched in here so its more useful

Reply:
will do mr BARON-PAELEN